README.md

```diff
@@ -115,6 +119,50 @@ Commit and push these changes to your local clone on GitHub, and then [create a
 
 We will be notified automatically when you have created your pull request, and providing that everything adheres to nf-core guidelines we will endeavour to approve your pull request as soon as possible.
 
+## Adding a new pipeline specific config
+
+Follow the previous guidelines to create a nf-core/configs, update the pipeline specific documentation, and create the necessary files.
```

Comment:
It might be better to be a bit more explicit here, as it is not exactly the same structure.

i.e. 

1. Define the pipeline specific parameters under `conf/pipeline/<your_pipeline>/<institution>.config`
2. Add a new entry to the pipeline specific config file under `pipeline/<your_pipeline>.config`
3. Add documentation to given `docs/pipeline/<your_pipeline>.md` describing the changes.

(And say more details below or something)